### PR TITLE
Get installables info from requirements.json

### PR DESCRIPTION
**Pull request by jrichter1**
Attempt to get all the names, versions and description from requirements.json instead of hardcoded strings in code.

### CODE REVIEW

**Comment by jrichter1** [comment]
Build is green, @dgolovin , @ppitonak , let me know what you think.

**Comment by dgolovin** [comment]
@jrichter1 thanks, it looks good, but I would postpone releasing this right now, because of conflicts.

**Comment by jrichter1** [comment]
I got it rebased for you :)

File: browser/model/installable-item.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
 'use strict';
 
+let path = require('path');
+let reqs = require(path.resolve('./requirements.json'));
```
Comment (dgolovin) [inline browser/model/installable-item.js]: this line only works for tests, it doesn't during installation.

Comment (dgolovin) [inline browser/model/installable-item.js]: Here is snippet that works https://github.com/redhat-developer-tooling/developer-platform-install/blob/master/browser/main.js#L79

**Comment by jrichter1** [comment]
I just found out I've pushed a wrong commit here (whoops). Anyway, I got that fixed and rather than using the horrible piece of path resolving again I put it into a method.

**Comment by dgolovin** [comment]
@jrichter1 could you rebase it one more time on top of all the changes?
Thanks!

**Comment by jrichter1** [comment]
Looks like I've messed up somewhere, investigating.

**Comment by jrichter1** [comment]
Rebased, mistakes mended.

**Pull request merged**